exdb/tests/integration_tests.py

```diff
@@ -375,6 +422,64 @@ def test_get_user(self):
         )
 
 
+class EditViewTest(StandardTestCase):
```

Comment:
Following our haphazard naming convention, this should probably be named `EditExperienceViewTest`.